_directory/data/danielspringer/data.yml

```diff
@@ -0,0 +1,29 @@
+---
+# name (required)
+# may be your real name or your nick; not your company name.
+# teams: feel free to add multiple entries for each of your members!
+name: Daniel Springer
+
+# geo location (required)
+# provide your favored level of detail by number of digits (like 50.107811 vs 50.11)
+# you may use services like openstreetmap.org to find out your geo location
+latitude: 48.7797306
+longitude: 9.179145199999994
+
+# bio
+# describe yourself, your involvement in the project and the community or whatever is relevant for you.
+# should be 300 chars at maximum. no markup or line breaks!
+bio: "I am Daniel from Stuttgart. I studied Business Informatics and I work with Redaxo (3.2) since 2007."
+
+# image or gravatar
+# provide image file from current folder (like 'johndoe.jpg') or gravatar url (generated via https://3v4l.org/OXG7H)
+# should be square and about 200-300 px size!
+image: daniel-springer.jpg
```

Comment:
das bild selbst fehlt noch

Reply:
![daniel-springer](https://user-images.githubusercontent.com/16903055/33615282-082436b4-d9da-11e7-8380-c2dc766045ae.jpg)

Reply:
Sorry, Github-Neuling.. :/

Reply:
hey @danidsp, kein Problem, die Map ist ziemlich praktisch, um daran GitHub-Funktionen auszuprobieren. Du bist fast am Ziel, das Bild müsste nur noch als Datei mit in den PR. Aktuell steckt es nur im Kommentar. Du müsstest also das Bild nochmal committen. Kannst du einfach hinterherschieben, der PR bleibt solange geöffnet für weitere Änderungen.